dev/com.ibm.ws.jaxrs.2.0.cdi/bnd.bnd

```diff
@@ -44,7 +44,7 @@ instrument.classesExcludes: com/ibm/ws/jaxrs20/cdi/internal/resources/*.class
 	com.ibm.ws.adaptable.module;version=latest,\
 	com.ibm.ws.webcontainer;version=latest,\
 	com.ibm.ws.jaxrs.2.0.web;version=latest,\
-	com.ibm.websphere.javaee.cdi.1.2;version=latest,\
+	com.ibm.websphere.javaee.cdi.2.0;version=latest,\
```

Comment:
This shouldn't be necessary.  We need to depend on the 1.2 version for JAX-RS 2.0 since JAX-RS 2.0 and CDI 1.2 are both part of EE7.